modules/concept-docs/pages/n1ql-query.adoc

```diff
@@ -7,42 +7,33 @@
 [abstract]
 {description}
 
-include::6.5@sdk:shared:partial$n1ql-queries.adoc[tag=intro]
+include::7.0@sdk:shared:partial$n1ql-queries.adoc[tag=intro]
 
 
 // Prepared Statements for Query Optimization
-include::6.5@sdk:shared:partial$n1ql-queries.adoc[tag=prepared]
+include::7.0@sdk:shared:partial$n1ql-queries.adoc[tag=prepared]
 
-NOTE: In Go SDK 2.1 and earlier, the `adhoc` parameter is `Adhoc`. 
+NOTE: In Go SDK 2.3 and earlier, the `adhoc` parameter is `Adhoc`.
 It is set to `false` by default -- set to `true` for a plan _not_ to be prepared, or a prepared plan _not_ to be reused.
-The 5000 limit to the client-side cache also does not apply. 
+The 5000 limit to the client-side cache also does not apply.
 
-[source,golang]
-----
-	query := "SELECT x.* FROM `travel-sample` x LIMIT 10;"
-	rows, err := cluster.Query(query, &gocb.QueryOptions{
-		Adhoc: false,
-	})
-	if err != nil {
-		panic(err)
-	}
-
-	for rows.Next() {
-	}
+QueryResult result = cluster.query(
```

Comment:
Is this meant to be here?

Reply:
Nope! Silly mistake, I must have pasted that in by accident. Good spot, will remove this.